test/models/lookup_keys/puppetclass_lookup_key_test.rb

```diff
@@ -9,6 +9,14 @@ class PuppetclassLookupKeyTest < ActiveSupport::TestCase
     refute lookup_key.valid?
   end
 
+  test "default_value is only validated if omit is true" do
```

Comment:
```suggestion
  test "default_value is only validated if omit is false" do
```
Please correct me if I am missing anything.